.github/workflows/ci.yml

```diff
@@ -0,0 +1,87 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # Test the latest stable release
+          - php-version: '7.1'
+            phpunit-version: 7
+          - php-version: '7.2'
+            phpunit-version: 8
+          - php-version: '7.3'
+            phpunit-version: 8
+          - php-version: '7.4'
+            phpunit-version: 8
+          - php-version: '8.0'
+            phpunit-version: 9
+          # Test Symfony LTS versions. Read more at https://github.com/symfony/lts
+          - php-version: '7.4'
+            dependencies: 'symfony/lts:^3'
+            phpunit-version: 8
+          - php-version: '8.0'
+            phpunit-version: 9
+            symfony-version: '^4'
+          - php-version: '8.0'
+            phpunit-version: 9
+            symfony-version: '^5'
+          - php-version: '8.0'
+            phpunit-version: 9
+            symfony-version: '^6'
+          # Minimum supported dependencies with the oldest PHP version
+          - php-version: '7.1'
+            phpunit-version: 7
+            composer-flag: '--prefer-stable --prefer-lowest'
+          # Test latest unreleased versions
+          - php-version: '8.0'
+            phpunit-version: 9
+            symfony-version: '^6'
+            stability: 'dev'
```

Comment:
This is a lot of versions. I just check on https://www.php.net/supported-versions.php, and I realized that versions below 7.4 are no more supported, so I don't see good reasons to still tests them. What about saving some CPU cycles and to support only the official versions: 7.4, 8.0 and 8.1
What do you think?

Reply:
i was thinking that this being an infrastructure tool, and not taking any extra effort to support old versions, we could keep supporting them so users can still upgrade RMT on legacy code bases.

but they can also keep installing the last released version if we drop support for old php versions. i let you decide.

the thing i find important is that composer.json matches with what we test - if we no longer test with < 7.4, we should also not have composer.json allow installation.

Reply:
You made a point here. And as it's already done. Let's keep it fort now